pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.ruff]
+select = [
+  "A",    # flake8-builtins
+  "ARG",  # flake8-unused-arguments
+  "B",    # flake8-bugbear
+  "C4",   # flake8-comprehensions
+  "C90",  # McCabe cyclomatic complexity
+  "E",    # pycodestyle
+  "EM",   # flake8-errmsg
+  "EXE",  # flake8-executable
+  "F",    # Pyflakes
+  "G",    # flake8-logging-format
+  "I",    # isort
+  "ICN",  # flake8-import-conventions
+  "INT",  # flake8-gettext
+  "ISC",  # flake8-implicit-str-concat
+  "PGH",  # pygrep-hooks
+  "PIE",  # flake8-pie
+  "PL",   # Pylint
+  "PT",   # flake8-pytest-style
+  "PYI",  # flake8-pyi
+  "RET",  # flake8-return
+  "RSE",  # flake8-raise
+  "RUF",  # Ruff-specific rules
+  "S",    # flake8-bandit
+  "SIM",  # flake8-simplify
+  "SLF",  # flake8-self
+  "T10",  # flake8-debugger
+  "TCH",  # flake8-type-checking
+  "TID",  # flake8-tidy-imports
+  "W",    # pycodestyle
+  "YTT",  # flake8-2020
+]
+ignore = [
+  "B904",
+  "PLR2004",
+  "RET505",
+  "RUF005",
+  "S110",
+  "S701"
+]
+line-length = 223
```

Comment:
Guessing this was to get it passing, rather than a preferred line length like the old 250 value

Reply:
88 is the preferred value for Python code.  The goal is to set high water marks so that if a contributor wants to go beyond that level they will need to justify their actions in pull requests.  The same is true of `max-args`, `max-branches`, `max-complexity`, etc.